### DISCUSSION TITLE
Save states from my pc not coming over gracefully to the server

### DISCUSSION THREAD

**Discussion by uttambharadwaj** [mbround18/valheim-docker#288] [Help!]
I have a save file where I've gone through about 60 days that I want to copy over to the cloud server. I followed all the steps in the guide and was successfully able to get the server running. 

To move save files, the guide asks us to copy over the .db and .fwl files for the save and copy it into ./valheim/saves/worlds/, which I did, and then changed the name of the world in the docker compose yaml script. But this keeps making a new world and not actually bringing over the old one. I am not sure what I am doing wrong. Also this world seems to be a completely different seed, so it's not just restarting my seed at 0.

Help!

**Comment by CosmicHorrorDev** [comment]
Are there save files from the new world appearing with a new name?

**Reply by uttambharadwaj** [reply]
No they seem to be overwriting the old one. My  file is called HoesWorld, which is what I named both the .fwl and .db files that came in. They seem to be getting updated with the new saves though, overwriting the old worlds.

**Reply by uttambharadwaj** [reply]
I was also having an issue where it said something like incompatible version while loading the world in the script, which lead me to believe there was some issue with the save file, but I have confirmed that copying it and renaming it on my PC does take me to the same save file, so it's not a file issue.

I also tried putting the save directly in the steam/.../valheim/.../worlds folder, which seems to work as per the script, but still loads up the old files weirdly enough.

**Reply by CosmicHorrorDev** [reply]
So just to clarify, is loading up the old files meaning that it is using the save files you copied over?

**Reply by mbround18** [reply]
Is this potentially related to https://github.com/mbround18/valheim-docker/discussions/280 ?

**Reply by uttambharadwaj** [reply]
So I put in HoesWorld.db and HoesWorld.fwl to the save folder. When the script runs, I can see it say loading HoesWorld... but the file that it has loaded isn't my save file, it seems to be a completely different world. I'm not sure how to more specifically explain this.

**Reply by uttambharadwaj** [reply]
> Is this potentially related to #280 ?

Seems kinda like it yeah

**Reply by mbround18** [reply]
Would you mind sending me a zip of your worlds on discord? Click the discord release notifications link in the read me to join and than just dm me.
